content/en/user-guide/aws/docdb/index.md

```diff
@@ -231,16 +232,125 @@ the [MongoDB documentation](https://www.mongodb.com/docs/).
 ### Connect to DocumentDB using Node.js Lambda
 
 {{< alert title="Important" color="success">}}
-You need to set `DOCDB_PROXY_CONTAINER=1` when starting LocalStack to be able to use the `LOCALSTACK_HOSTNAME` in lambda to connect to the instance.
+You need to set `DOCDB_PROXY_CONTAINER=1` when starting LocalStack to be able to use the `localhost.localstack.cloud` in lambda to connect to the instance.
```

Comment:
we already explain it at the very beginning of the docs:

> Using the flag DOCDB_PROXY_CONTAINER=1 the default behavior changes and the container will be started as proxied container. Meaning a port from the [pre-defined port](https://localstack-docs-preview-pr-935.surge.sh/references/external-ports/) range will be chosen, and when using lambda, you can use LOCALSTACK_HOSTNAME to connect to the instance.

what else would you add here? :)

Reply:
Yeah, that's exactly what I was looking for 👍 . It might be better to move it here because it is mostly relevant in that context 🤷 .

We should also update the reference to `LOCALSTACK_HOSTNAME`, also in the config docs https://docs.localstack.cloud/references/configuration/#documentdb